evaluator.py

```diff
@@ -194,10 +272,21 @@ def evaluate_pass_fail_status(self) -> PassFailStatus:
                 total_passing += 1
```

Comment:
you may want to rename these to `tft_passing`, `tft_failing` to be consistent in this block:
```python
num_tft_passed=total_passing,
num_tft_failed=total_failing,
num_plugin_passed=plugin_passing,
num_plugin_failed=plugin_failing
```

Reply:
Done